### ISSUE TITLE
when use batch norm =1 ,  batchnorm is useless?

### ISSUE THREAD

**Issue by ZhangSanFengByGit** [foolwood/DaSiamRPN#51]
when use batch norm =1 ,  batchnorm is useless?
